### ISSUE TITLE
Error when compiling on Windows 64bit

### ISSUE THREAD

**Issue by biskitdev** [fyne-io/fyne#1255]
### Describe the bug:
\
When I try to compile an app using fyne, I get the error:

```
# fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.3/glfw     
cc1.exe: sorry, unimplemented: 64-bit mode not compiled in
# fyne.io/fyne/vendor/github.com/go-gl/gl/v3.2-core/gl    
cc1.exe: sorry, unimplemented: 64-bit mode not compiled in
```

### Example code:

```go
package main

import (
	"fyne.io/fyne/app"
	"fyne.io/fyne/widget"
)

func main() {
	app := app.New()

	w := app.NewWindow("Hello")
	w.SetContent(widget.NewLabel("Hello Fyne!"))

	w.ShowAndRun()
}
```

### Device (please complete the following information):
 - OS: Windows 64-bit
 - Version: 10
 - Go version: 1.15
 - Fyne version: Unknown

**Comment by biskitdev** [comment]
I fixed this by installing [mingw-w64 8.1.0](https://sourceforge.net/projects/mingw-w64/) almost by accident following a tutorial on SDL2

**Comment by biskitdev** [comment]
oops i forgot to close
